multi-node-k8s.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+set -e -o pipefail
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt | sed 's/^v//')"
```

Comment:
it's better to make it as an argument as we need to run E2E against multiple kubernetes versions parallelly